packages/applications-service-api/src/services/submission.service.js

```diff
@@ -0,0 +1,60 @@
+const db = require('../models');
+
+const PLACEHOLDER_SUBMISSION_ID = 0;
+
+const createSubmission = async (submission) => {
+	const submissionId = submission.submissionId || PLACEHOLDER_SUBMISSION_ID;
+
+	const dbSubmission = await db.Submission.create({
+		name: submission.name,
+		email: submission.email,
+		interestedParty: submission.interestedParty,
+		iPReference: submission.ipReference,
+		deadline: submission.deadline,
+		submissionType: submission.submissionType,
+		sensitiveData: submission.sensitiveData,
+		lateSubmission: submission.lateSubmission,
+		caseReference: submission.caseReference,
+		dateSubmitted: submission.dateSubmitted,
+		submissionId: submissionId,
+		representation: submission.representation
+	});
+
+	if (submissionId === PLACEHOLDER_SUBMISSION_ID) {
```

Comment:
The behaviour I saw when testing from an empty DB locally was that if the submissions table was empty, the first row should be given an ID of `1` rather than `0`.